mftf/2.3/reporting.md

```diff
@@ -0,0 +1,367 @@
+---
+---
+
+# Reporting
+
+The Magento Functional Testing Framework provides two types of reporting:
+
+- In CLI, when you run [`mftf`][mftf] or [`codecept`][codecept] commands.
+- In a web browser, using [Allure Framework][] after test run was complete.
+
+All the reporting artifacts are stored at the `dev/tests/acceptance/tests/_output` in the Magento root directory.
+The directory contains:
+
+- `allure-results/` that is a directory generated and served by the Allure Framework.
+- `failed` that is a text file containing relative paths to failed tests after the last test run.
+  The paths are relative to `dev/tests/acceptance/`.
+- `.html` and `.png` files that are screenshots of fails in HTML and PNG formats.
+  To cleanup the `_output/` directory, remove them manually.
+
+The `mftf` tool logs its output at `dev/tests/acceptance/mftf.log` continuously.
+
+## Command line
+
+The MFTF reports about its progress during test run when you run the `mftf` CLI tool with [`run:test`][] or [`run:group`][] commands.
+
+The report may contain three main parts:
+
+- Pre-run checks:
+  - Environment check, such as PHP warnings, etc.
+  - XML test validation like deprecation warnings such as missing required components in XML tests.
+- Codeception report which is the actual report about progress of each test.
```

Comment:
which is the progress report for each test ...

Reply:
:ok: